src/components/Select.vue

```diff
@@ -318,7 +318,7 @@
           :multiple="multiple"
           :disabled="disabled"
         >
-          <span class="selected-tag" v-bind:key="option.index">
+          <span class="selected-tag" v-bind:key="option.index" :class="{ 'multiple':multiple }">
```

Comment:
This particular change allows to better style a CSS for it. In my usage, I use "chips" for each multiple input, but not on regular inputs. To differentiate and style both, this is very useful.